great-idea/index.html

```diff
@@ -14,10 +14,96 @@
   <![endif]-->
 </head>
 
-  <body>
+<body class="container">
+  <nav>
+    <a href="services.html">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+  
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+
+  <header class="header">
+    <div class="col">
+      <h1>
+        Innovation<br>
+        On<br>
+        Demand
+      </h1>
+          
+      <button type="button">Get Started</button>
+    
+    </div>
+    <div class="col">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+    
+  
+  </header>
+
+<section class="features">
```

Comment:
Good use of your semantic elements, your line spacing, and indentation. You're building good habits already - keep this up!